### PR TITLE
Ruben

### PR DESCRIPTION
**Pull request by RubenDguez**
I have worked with the footer of the app, and other files please review and merge thanks

### CODE REVIEW

File: HotelYeah/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "axios": "^0.21.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-native": "^0.63.4",
```
Comment (CharlesBrough) [inline HotelYeah/package.json]: is there a reason this dep has been added? i thought react-native was for android/iOS apps

File: HotelYeah/src/App.tsx [hunk 2]
```diff
@@ -18,12 +19,17 @@ const App = () => {
       <ThemeProvider theme={theme}>
         <div className="App">
           <Navbar />
-          {/* Routing here for now */}
+          
 
-          <Routes />
-          {/* If user is signed in */}
-          <AuthRoutes />
+          <div style={{ minHeight: '100vh' }}>
```
Comment (CharlesBrough) [inline HotelYeah/src/App.tsx:24]: could probably refactor this to App.css

**Review by CharlesBrough — COMMENTED** [review]
just a couple comments, one question, otherwise it looks good

File: HotelYeah/package.json (hunk 1 above)
Comment (RubenDguez) [reply HotelYeah/package.json]: no reason for this one, my bad, I forgot to remove it...
It should be fixed now

**Pull request merged**